### DISCUSSION TITLE
reordering columns in vaex

### DISCUSSION THREAD

**Discussion by Hasham04** [vaexio/vaex#1913] [Q&A] [answered]
I am a beginner using vaex trying to solve some data problems. I had a query and was looking for some suggestions. Any help is greatly appreciated. So my question is how do I reorder columns in vaex. for example, I want the 5th column at number 1 and the first column at number 5, etc. I know we can use the reindex method in pandas, is there a way to mimic that in vaex. thanks for your help.

**Comment by maartenbreddels** [comment, marked as answer]
Hi Hasham!

you can do this using:
```
df1 = df[['a', 'b', 'c']]
df2 = df[['b', 'a', 'c']]
```
Does that make sense?

Regards,

Maarten

**Reply by Hasham04** [reply]
so basically reorder the column names inside a list the way I want them and pass them to the data frame.

**Reply by Hasham04** [reply]
maybe I am still a little confused. sorry

**Reply by maartenbreddels** [reply]
Yes, that's it!
